java-manta-client/src/main/java/com/joyent/manta/http/EncryptionHttpHelper.java

```diff
@@ -670,7 +708,7 @@ private void attachMetadata(final HttpResponse response) {
      *
      * @param metadata metadata to append additional values to
      */
-    private void attachEncryptionCipherHeaders(final MantaMetadata metadata) {
+    public void attachEncryptionCipherHeaders(final MantaMetadata metadata) {
```

Comment:
I made these public as the least bad expedient option.  If users are not expected to interact with the helpers directly (which I believe is the case), maybe having these public is okay.